wordfreq_builder/wordfreq_builder/word_counts.py

```diff
@@ -17,9 +17,14 @@ def count_tokens(filename):
     """
     counts = defaultdict(int)
     with open(filename, encoding='utf-8', errors='replace') as infile:
-        for line in infile:
-            for token in simple_tokenize(line.strip()):
-                counts[token] += 1
+        if lang == 'ar':
+            for line in infile:
+                for token in simple_tokenize(line):
+                    counts[standardize_arabic(token)] += 1
```

Comment:
The  Arabic standardizer does operated before tokenization. However, we also have to standardize over data source that are not parsed by the wordfreq tokenizer (e.g. Leeds and open-subtitles). I am not sure how much time is saved by having the lang == 'ar' check outside the loop. However, for English, it is the difference between running the check once vs over 6,000,000 times.